docs/use_cases/model_applications/s2s/SpaceTimeCrossSpectra.py

```diff
@@ -0,0 +1,133 @@
+"""
+UserScript: Make a Hovmoeller plot 
+========================================================================
+
+s2s/SpaceTimeCrossSpectra
+cross-spectra.py
+
+"""
+
+#################################################################################
+# Scientific Objective
+# --------------------
+#
+# This use case calls the METplotpy space time plot to create a sample cross 
+# spectra diagram
+# using sample data created by METcalcpy hovmoeller functions
+#
+# The space time plot and cross spectra calculations were created by Maria Gehne 
+# at the 
+# Physical Sciences Labratory in NOAA 
+
+#################################################################################
+# Datasets
+# --------
+#
+
+##############################################################################
+# METplus Components
+# ------------------
+#
+# This use case runs the UserScript wrapper tool to run a user provided script,
+# in this case, cross-spectra.py.
+#
+
+##############################################################################
+# METplus Workflow
+# ----------------
+#
+# This use case does not loop but plots the entire time period of data
+# 
+# UserScript
+#
+#
+
+##############################################################################
+# METplus Configuration
+# ---------------------
+#
+# METplus first loads all of the configuration files found in parm/metplus_config,
+# then it loads any configuration files passed to METplus via the command line
+# with the -c option, i.e. -c parm/use_cases/model_applications/s2s/hovmoeller_diagram.conf
+#
+# .. highlight:: bash
+# .. literalinclude:: ../../../../parm/use_cases/model_applications/s2s/SpaceTimeCrossSpectra/cross-spectra_diagram.conf
+#
+
+#############################################################################
+# MET Configuration
+# ---------------------
+#
+# There are no MET tools used in this use case.
+#
+
+##############################################################################
+# Python Embedding
+# ----------------
+#
+# There is no python embedding in this use case
+#
+
+##############################################################################
+# Running METplus
+# ---------------
+#
+# This use case can be run two ways:
+#
+# 1) Passing in hovmoeller_diagram.conf, 
+# then a user-specific system configuration file::
+#
+#        run_metplus.py \
+#        -c /path/to/METplus/parm/use_cases/model_applications/s2s/hovmoeller_diagram.conf \
```

Comment:
Should this be hovmoeller conf file or the UserScript_obsPrecip_obsOnly_CrossSpectraPlot.conf?